src/ElasticSearchBase.cpp

```diff
@@ -28,19 +28,24 @@ std::string ElasticSearchBase::getElasticSearchUrlonIndex(std::string index) {
   return str;
 }
 
-std::string ElasticSearchBase::getElasticSearchUrlOnDoc(std::string index, Int64 doc) {
+std::string ElasticSearchBase::getElasticSearchUrlOnDoc(std::string index, std::string docType, Int64 doc) {
   std::stringstream out;
-  out << getElasticSearchUrlonIndex(index) << "/" << DEFAULT_TYPE << "/" << doc;
+  out << getElasticSearchUrlonIndex(index) << "/" << docType << "/" << doc;
```

Comment:
same as before we should keep the DEFAULT_TYPE and don't add it as a parameter

Reply:
fixed